tasks.py

```diff
@@ -462,6 +462,10 @@ def kill_node(c, node=None, all_=False):
         cmd = (
             f"pid=$(ps aux | grep '[c]elery' | grep 'worker' | grep '{node_pattern}' | awk '{{print $2}}') "
             f"&& pgrep -P $pid | xargs kill -9 "
+        )
+        run(c, cmd, warn=True, show_ok=False)
+        cmd = (
+            f"pid=$(ps aux | grep '[c]elery' | grep 'worker' | grep '{node_pattern}' | awk '{{print $2}}') "
```

Comment:
you could put a comment here explaining this

Reply:
There is no change in what we want / what we do, it was just a bug in the previous logic.

When a `node_pattern` was not used there was not "celery parent process" so the 2nd command exited with error and didn't run the final command.

I added some new commends nonetheless to make it a bit more clear.